src/engine/MILPEncoder.cpp

```diff
@@ -187,3 +203,152 @@ void MILPEncoder::encodeMaxConstraint( GurobiWrapper &gurobi, MaxConstraint *max
     }
     _binVarIndex++;
 }
+
+void MILPEncoder::encodeSigmoidConstraint( GurobiWrapper &gurobi, SigmoidConstraint *sigmoid )
+{
+    unsigned sourceVariable = sigmoid->getB();  // x_b
+    unsigned targetVariable = sigmoid->getF();  // x_f
+    double sourceLb = _tableau.getLowerBound( sourceVariable );
+    double sourceUb = _tableau.getUpperBound( sourceVariable );
+
+    if ( sourceLb < 0 && sourceUb > 0 )
```

Comment:
Use FloatUtils::gt() and FloatUtils::lt() instead of <, > 
https://github.com/NeuralNetworkVerification/Marabou/blob/9c08e9cc7ca45fd8dba43fff0e2ab3bd5546dd0f/src/common/FloatUtils.h#L89